src/mcp/tools.cpp

```
@@ -0,0 +1,1258 @@
+            if (trimmed_end != std::string::npos) {
+                current_block.instructions.push_back(inst_text.substr(0, trimmed_end + 1));
+            }
+        }
+    }
+    if (!current_block.instructions.empty()) {
+        blocks.push_back(std::move(current_block));
+    }
+
+    // Build label → PC map.
+    std::map<std::string, prevail::Label> label_map;
+    int pc = 0;
```

Clang (14.0.6)

[warning] variable name 'pc' is too short, expected at least 3 characters

(readability-identifier-length)